frontend/packages/dev-console/src/components/pipelines/pipeline-resource/PipelineGitOptions.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { TextInputTypes } from '@patternfly/react-core';
+import { InputField } from '../../formik-fields';
+
+const PipelineGitOptions: React.FC = () => (
+  <React.Fragment>
+    <InputField
+      type={TextInputTypes.text}
+      name="params.url"
+      label="URL"
+      helpText="Please provide git URL."
+      required
+    />
+    <InputField
+      type={TextInputTypes.text}
+      name="params.revision"
+      label="Revisions"
```

Comment:
According to design doc:
```suggestion
      label="Revision"
```